graylog2-web-interface/src/views/logic/widgets/Widget.ts

```diff
@@ -35,14 +38,17 @@ type DeserializesWidgets = {
   fromJSON: (value) => Widget;
 };
 
+const isNullish = (o: any) => (o === null || o === undefined);
+
 class Widget {
   _value: WidgetState;
 
   // eslint-disable-next-line @typescript-eslint/no-use-before-define
   static Builder: typeof Builder;
 
-  constructor(id: string, type: string, config: any, filter?: string, timerange?: TimeRange, query?: QueryString, streams: Array<string> = []) {
-    this._value = { id, type, config, filter: filter === null ? undefined : filter, timerange, query, streams };
+  // eslint-disable-next-line default-param-last
+  constructor(id: string, type: string, config: any, filter?: string, timerange?: TimeRange, query?: QueryString, streams: Array<string> = [], filters?: FiltersType) {
```

Comment:
Afaics we do not rely on the default value for the `streams` parameter, so we can just remove it. This way we do not need to suppress `default-param-last` for the `filters` parameter.